packages/cascara/src/ui/Form/Form.fixture.js

```diff
@@ -122,15 +116,44 @@ const BasicForm = (props) => {
   );
 };
 
+const DataWithDisplay = (props) => (
+  <>
+    <h1>Data with display</h1>{' '}
+    <p>A form can not be in an editing state initially.</p> <Form {...props} />
+  </>
+);
+
 const InitialEditing = (props) => (
   <>
     <h1>Initial Editing Form</h1>{' '}
     <p>A form can be in an editing state initially.</p> <Form {...props} />
   </>
 );
 
+const LoadingForm = (props) => (
+  <>
+    <h1>Loading state form</h1>{' '}
+    <p>A form can show a loader while the default values are rendered.</p>{' '}
+    <Form {...props} />
+  </>
+);
+
+const EmptyForm = (props) => (
+  <>
+    <h1>Empty</h1>
+    <p>
+      If a form does not receive dataDisplay prop as an array with values, the
+      Form component will notify there is not data to present.
+    </p>{' '}
+    <Form {...props} />
+  </>
+);
 export default {
-  basic: <BasicForm actions={actions} data={data} dataDisplay={dataDisplay} />,
+  dataOnly: <BasicForm actions={actions} data={{}} dataDisplay={dataDisplay} />,
+  dataWithDisplay: (
+    <DataWithDisplay actions={actions} data={data} dataDisplay={dataDisplay} />
+  ),
+  empty: <EmptyForm actions={actions} data={data} dataDisplay={[]} isLoading />,
```

Comment:
```suggestion
  initialEditing: <EmptyForm actions={actions} data={data} dataDisplay={[]} isInitialEditing />,
```

Technically, I think this fixture will change to use isInitialEditing. We probably want to rename it for initial editing as well.

We may also want to add one more fixture for showing data empty (isLoading) and not setting isInitialEditing and make sure it shows the warning we expect.

Reply:
Status added:
dataOnly, dataWithDisplay,empty, initialEditing,loading